samples/xti/codereadercallback.h

```diff
@@ -1,4 +1,4 @@
-/////////////////////////////////////////////////////////////////////////////
+﻿/////////////////////////////////////////////////////////////////////////////
```

Comment:
Right. I wonder how that happened, and how could it be made more obvious in the diff.
Should be fixed now.